zanata-war/src/main/webapp/WEB-INF/layout/admin/contact_admin_modal.xhtml

```diff
@@ -32,10 +34,26 @@
       return getContactAdminForm().find('#contact-admin-send-button');
     }
 
+    // Returns a span.
+    function getContactAdminMessageErrorField(){
+      return getContactAdminForm().find('#contactAdminForm\\:messageField\\:input\\:contact-admin-error-message\\:contactAdminForm\\:messageField\\:input\\:contact-admin-message');
+    }
+
     function toggleSendButton() {
       var hasMessage = getContactAdminMessageField().val() !== '';
       getContactAdminSendButton().prop('disabled', !hasMessage);
     }
+
+    function completeSending() {
+      var errMsg = getContactAdminMessageErrorField().text();
+      if (errMsg != '') {
```

Comment:
I tried to control visibility of the span. 
element.is(":disabled")  element function doesn't work for a span. It returns "false" in both cases.
It seems that everything is fine with visibility check.